jx_sqlite/aggs_table.py

```diff
@@ -528,6 +532,14 @@ def _groupby_op(self, query, frum):
                   "\nFROM\n" + quote_table(self.sf.fact) + " " + nest_to_alias["."] + \
                   "\nWHERE\n" + where + \
                   "\nGROUP BY\n" + ",\n".join(groupby)
+        
+        if query.sort:
+            command += "\nORDER BY " + ",\n".join(
+                "(" + sql[t] + ") IS NULL"  + ",\n" +
+                sql[t] + (" DESC" if s.sort == -1 else "")
+                for s, sql in [(s, s.value.to_sql(schema)[0].sql) for s in query.sort]
+                for t in "bns" if sql[t]
```

Comment:
Good list comprehension!  Next time, instead of 540, 541, write a single line:

    for s in query.sort for t, sql in s.value.to_sql(schema)[0].sql.items()